torchbenchmark/__init__.py

```diff
@@ -2,7 +2,6 @@
 from pathlib import Path
 import subprocess
 import sys
-import torch
```

Comment:
Remove the torch import here because it is not needed and will create problem if we import torchbenchmark without torch installed. In this case, torchbenchmark is imported at initialization, where torch is not yet installed.